autoemulate/emulators/neural_net_torch.py

```diff
@@ -106,5 +164,73 @@ def get_grid_params(self, search_type="random"):
 
         return param_grid
 
+    def __sklearn_is_fitted__(self):
+        return hasattr(self, "n_features_in_")
+
     def _more_tags(self):
-        return {"multioutput": True}
+        return {"multioutput": True, "poor_score": True}
```

Comment:
Adding the `_xfail_checks` tag here allows to skip tests in `@parametrize_with_checks` (see [custom estimators](https://scikit-learn.org/stable/developers/develop.html)) and name a reason. Would be good to add the four failing tests so that the CI runs.